SECURITY.md

```diff
@@ -1,18 +1,38 @@
-# Security Policy
```

Comment:
Heading should be `Security Policy`, `Coordinated Vulnerability Disclosure` should be a sub-heading, maybe even a sub-heading to `Reporting a suspected vulnerability`.

Perhaps we should also consider moving `Supported Versions` to the top.

Reply:
@stianst does it sound any better now? If not, would you like to make those changes?